### PR TITLE
Add initializer_list based WellContainer constructor

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/simulators/wells/WellContainer.hpp [hunk 1]
```diff
@@ -44,6 +45,15 @@ template <class T>
 class WellContainer {
 public:
 
+    WellContainer() = default;
+
+
+    WellContainer(const std::initializer_list<std::pair<std::string,T>>& init_list) {
```
Comment (bska) [inline opm/simulators/wells/WellContainer.hpp:51]: The typical way to pass an `initializer_list` to a function is by value as in
```C++
WellContainer(std::initializer_list<...> args)
```
There's nothing to gain by using a reference to `const` in this case.

Comment (joakim-hove) [reply opm/simulators/wells/WellContainer.hpp]: OK - value used

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**